ui/src/app/views/confirm-playground-mode-modal.tsx

```diff
@@ -0,0 +1,68 @@
+import * as Cookies from 'js-cookie';
+import * as React from 'react';
+
+import {Button} from 'app/components/buttons';
+import {CheckBox} from 'app/components/inputs';
+import {Modal, ModalBody, ModalFooter, ModalTitle} from 'app/components/modals';
+import colors from 'app/styles/colors';
+
+export interface Props {
+  onCancel: Function;
+  onContinue: Function;
+}
+
+interface State {
+  checked: boolean;
+}
+
+export class ConfirmPlaygroundModeModal extends React.Component<Props, State> {
+
+  public static DO_NOT_SHOW_AGAIN = 'DO_NOT_SHOW_AGAIN';
+
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      checked:  Cookies.get(ConfirmPlaygroundModeModal.DO_NOT_SHOW_AGAIN) === String(true)
```

Comment:
Couple of quick drive-by points here:

1) Shouldn't the cookie string (currently DO_NOT_SHOW_AGAIN) have some prefix that clarifies that this is unique to the confirm playground modal component?

2) We do have some sort of server-side storage that might be shifted to support this case as well. We already use it to prevent showing the quick tour modal to a user after they've seen it. See https://github.com/all-of-us/workbench/blob/master/api/src/main/resources/workbench.yaml#L313

I was running into this API call while working on the homepage recently, and I thought with the right client-side mapping it could be a nice way of tracking certain small bits of user-centric state like this. Just food for though.

Reply:
1. Good point, I'll change that to something more unique.

2. Cool! I didn't know about that. I think I'm inclined to stick with the cookies solution for now since its simpler and Karthik since its fine from a UX perspective. But I'll let him know there's a reasonable path forward to implement this with back end persistence.